src/Filter/AbstractDateFilter.php

```diff
@@ -45,23 +41,16 @@ final public function getDefaultOptions(): array
         ];
     }
 
-    final public function getRenderSettings(): array
+    /**
+     * @return array<string, mixed>
```

Comment:
do we need this? shouldnt be on interface?

Reply:
On the interface it's just a @method (adding the method sould be Bc break) so we cannot add phpdoc on the interface